src/components/AsideHeader/components/CompositeBar/Item/Item.module.scss

```
@@ -214,31 +332,63 @@ $actionLeftRightMargin: 10px;
+            var(--gn-aside-header-item-icon-size, 18px) + var(--g-spacing-2)
+        );
+        --gn-aside-header-popup-item-title-gap: var(--g-spacing-2);
+        --_--popup-item-radius: var(
+            --gn-aside-header-item-expanded-radius,
+            #{variables.$item-expanded-radius}
+        );
+        --_--border-radius: calc(var(--_--popup-item-radius) + var(--_--popup-padding));
+        --g-popup-border-radius: var(--_--border-radius);
+
+        @include floating-surface-box-shadow-default;
         display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)